src/dos/drive_local.cpp

```diff
@@ -568,7 +568,23 @@ localDrive::localDrive(const char* startdir, uint16_t _bytes_sector,
 	dirCache.SetBaseDir(basedir);
 }
 
-bool localFile::Read(uint8_t *data, uint16_t *size)
+static DiskType getDiskTypeFromMediaByte(uint8_t media_byte)
```

Comment:
We just use our standard new naming scheme when creating new functions, regardless of what the file uses, so `get_disk_type_from_media_byte`. The idea is we'll convert everything we touch over time to the new style.